diff --git a/shared/origami/python/origami_module.cpp b/shared/origami/python/origami_module.cpp
index a85c5da..c4b9b07 100644
--- a/shared/origami/python/origami_module.cpp
+++ b/shared/origami/python/origami_module.cpp
@@ -154,6 +154,7 @@ NB_MODULE(origami, m) {
                           size_t,
                           std::tuple<double, double, double>>())
       .def("print", &hardware_t::print)
+      .def_rw("arch", &hardware_t::arch)
       .def_rw("N_CU", &hardware_t::N_CU)
       .def_rw("lds_capacity", &hardware_t::lds_capacity)
       .def_rw("mem1_perf_ratio", &hardware_t::mem1_perf_ratio)
